src/thermo/ThermoFactory.cpp

```diff
@@ -53,56 +53,56 @@ ThermoFactory::ThermoFactory()
     addAlias("none", "ThermoPhase");
     addAlias("none", "None");
```

Comment:
Default is lowercase `none` here.

above, it may be worth deprecating `ThermoPhase` as well.